setup.py

```diff
@@ -3,9 +3,18 @@
 import os
 
 from setuptools import setup, Extension, find_packages
-from nml import version_info
+from setuptools.command.build_py import build_py
+from nml import parser, version_info
 NML_VERSION = version_info.get_and_write_version()
 
+
+class NMLBuildPy(build_py):
+    def run(self):
+        # Create a parser so that nml/generated/{parse,lex}tab.py are generated.
```

Comment:
```suggestion
        from nml import parser
        # Create a parser so that nml/generated/{parse,lex}tab.py are generated.
```

Reply:
Thanks. This fix was what I thought of when I saw your last comment, but I hadn't got to it yet. :P